greatNovels.sql

```diff
@@ -0,0 +1,161 @@
+CREATE DATABASE greatNovels;
```

Comment:
You can delete this file. We used .sql files as a stand in until we learned migrations. In the real world you'd use migrations from the start and never have .sql files.